openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -32,6 +32,62 @@ function getJqueryElements(selector){
     return jQueryElementArray;
 }
 
+export function initEditionEditPage() {
+    $(function () {
```

Comment:
@jdlrobson I don't believe we need this, right? Everything in index.js is loaded after document.ready, no?

Reply:
Correct, jQuery.ready should only be used in the index.js file.